force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js

```diff
@@ -20,68 +20,63 @@ export default class RecurringDonationTable extends LightningElement {
     
     data = [{
```

Comment:
I know it's unrelated to the change, but is this test data? If so, why is it in the component itself?

Reply:
Actually, they are only design data, they are not intended to be used in tests.

Reply:
What do you mean by design data? Is it like an example use case that will be displayed to the end-user?

Reply:
That's right @andrewyu-salesforce , sorry for the confusion, it's mock data, but it's not currently being used for testing, just to simulate the data and how the user should see it.
That is why it is not in a separate file or in the folder corresponding to the test for LWC.